README.md

```diff
@@ -203,7 +204,11 @@ Build the memcached-operator image and push it to your registry. The following e
 ```sh
 $ operator-sdk build quay.io/<user>/memcached-operator:v0.0.1
 $ sed -i 's|REPLACE_IMAGE|quay.io/<user>/memcached-operator:v0.0.1|g' deploy/operator.yaml
-# On OSX use:
+```
+
+**NOTE** On OSX use:
+
+```sh
```

Comment:
This change makes it seem like `docker push` should only be run on OSX. 

The Go user guide has a better way: https://github.com/operator-framework/operator-sdk/blame/master/doc/user-guide.md#L237-L248

Reply:
Done.